example/src/main/deploy/openshift/common.sh

```diff
@@ -11,6 +11,22 @@
 #    Red Hat - initial creation
 #    Bosch Software Innovations GmbH
 
+function prepare_openshift {
+    # creating the directory for Hono Server persistent volume
+    if [ ! -d /tmp/hono ]; then
+        mkdir /tmp/hono
+        chmod 777 /tmp/hono
+    else
+        echo /tmp/hono already exists!
+    fi
```

Comment:
this whole code block is useless FMPOV because it creates the directory on the local host and *not* on the OpenShift node, where it needs to be created if we want to use a host path PV (which we do). I have added this required (manual) step to the [openshift deployment guide](https://www.eclipse.org/hono/deployment/openshift/#minishift) already.

Reply:
the problem here is that we either can assume that we are using minishift and then simply add the `minishift ssh` command to this script, or we keep this script open to whatever openshift system you are using but then we need to point out that you need to provision a volume on openshift yourself ...

Reply:
I agree. I removed it. Maybe we can suggest using `host-pv-dir` minishift option as described here

https://docs.openshift.org/latest/minishift/using/basic-usage.html#persistent-volumes

I'll try it out.